DataFormats/FEDRawData/doc/FEDRawData.doc

```diff
@@ -15,6 +15,72 @@
 Classes to store raw data blocks in the event.
 FED data blocks are required to have correct padding (64 bit).
 
+<!-- This table must be created with DataFormats/FEDRawData/doc/createFEDtable.py -->
```

Comment:
changes in this file appear inappropriate for 80X.
some of the lines below contain references to classes that do not exist in 80X.
You can re-generate this table as suggested in the comment